### ISSUE TITLE
How to choose keys.txt as a vocabulary ? for example where can i find english vocab.

### ISSUE THREAD

**Issue by karimcossentini** [wenwenyu/PICK-pytorch#88]

**Comment by ducviet00** [comment]
You can iterate though all your training dataset and get a set of character, include the space.
They encode by the character index, and vocab size in Embedding Layer equal to the len of keys. 
I think it is not good to compare with new method like BPE encoding

**Comment by babyhockey** [comment]
Hi ducviet00, is the vocabulary a set of characters or words? If it is a set of characters, does it mean we just list "abcd...zABCD...Z" plus numbers and special chars in keys.txt? Thanks!

**Comment by karimcossentini** [comment]
Hi babyhockey, it's a good question cuz I have a problem with this vocab file , did you find the answer ?

**Comment by babyhockey** [comment]
Yes, it's a list of English characters plus numbers and special characters.

**Comment by karimcossentini** [comment]
The default vocab file (keys.txt) in this repo is in chinese , I translated it and I noticed that it contains not only characters , but sentences and words etc... so I did not understand what actually this file is

**Comment by babyhockey** [comment]
As far as I can tell, the default file contains a list of Chinese
characters.

**Comment by arunmack789** [comment]
> Hi ducviet00, is the vocabulary a set of characters or words? If it is a set of characters, does it mean we just list "abcd...zABCD...Z" plus numbers and special chars in keys.txt? Thanks!

for custom dataset how to write keys.txt file
